### PR TITLE
Write lesson to demonstrate debugging in Spring

### PR DESCRIPTION
**Pull request by kcleland0818**
This change is being made in the consumer canvas per the conversation on 7/22 to keep the same modules for Blackrock and AWS to streamline.

- Demonstrated using the debugger.

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]
This will be super helpful for the students!

**Comment by kcleland0818** [comment]
@developerjaym - I had to rewrite this lesson. I mistakenly assumed the "acutator" tool I was using was also in the free community edition of IntelliJ. Turns out it is limited to the Ultimate edition!

@linda-seiter - Also, IntelliJ only warns about Spring mistakes in the Ultimate edition too; not the community edition. So I have students stepping through the issue with a path variable and provided screenshots of other common issues as you mentioned.

Long story short: I just provided advertising for the Ultimate edition of IntelliJ.

File: README.md [hunk 1]
```diff
@@ -2,31 +2,189 @@
...
+
+## Conclusion
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+Seeing what is sent to the application and what it is getting back from the
+data source is incredibly helpful when trying to figure out the unexpected. The
+debugger tool in IntelliJ is not to be overlooked and can serve as an
+efficient and quick way to figure out what may be wrong. Using this in
+conjunction with Postman and the Java Visualizer tool will help in debugging
```
Comment (developerjaym) [inline README.md:188]: Do you still mention the Java Visualizer elsewhere?

**Review by linda-seiter — APPROVED** [review]
Looks great! This will be really helpful for the students!

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md:188]: I added a clarification that, even though the Java Visualizer wasn't used here, students should still be encouraged to use it when appropriate if it helps. Assumption is that some of the lessons from previous modules will include how to use the Java Visualizer in more detail.

**Review by linda-seiter — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]
